src/modules/App/App.js

```diff
@@ -5,18 +5,47 @@ import { appReducer, initialState } from "../common/appReducer";
 import "./App.css";
 import MissionControl from "../views/MissionControl";
 import StyleSample from "../views/StyleSample";
-import Button from '../../ui/button/Button'
+import Button from '../../ui/button/Button';
+import DailyMission from "../views/DailyMission";
+import Sidebar from "../common/sidebar/Sidebar";
 
 function App() {
   const [state, dispatch] = useReducer(appReducer, initialState);
   const [show, setShow] = useState(true)
 
   return (
     <AppContext.Provider value={[state, dispatch]}>
-      <div className="App">
-        <Button onClick={() => setShow(!show)}>{show ? "Hide Style Guide" : "Show Style Guide"}</Button>
-        {show && <StyleSample />}
-      </div>
+      <main className="App">
+        <Sidebar />
+        <div className='routes'>
+          <Switch>
+            <Route
```

Comment:
Thanks for adding some routes. Let's start building out those components! SideBar, comin' up.